docs/onboarding-guide.md

```diff
@@ -0,0 +1,171 @@
+
+# Onboarding Guide for .NET Deployment Tool
+
+**AWS Deploy Tool for .NET** is an interactive tooling for the .NET CLI and the AWS Toolkit for Visual Studio that helps deploy .NET applications with minimum AWS knowledge, and with the fewest clicks or commands.
+
+## Reference Links
+
+* Repository - https://github.com/aws/aws-dotnet-deploy
+* Public documentation - https://aws.github.io/aws-dotnet-deploy/
+
+## Using the Deployment Tool from Source
+
+* Ensure that all the [pre-requisties](https://aws.github.io/aws-dotnet-deploy/docs/getting-started/pre-requisites/) (including Docker) are installed and your default AWS profile has admin access
+* Clone the repository using `git clone https://github.com/aws/aws-dotnet-deploy.git`
+* Open `AWS.Deploy.sln` in Visual Studio.
+* Ensure that `../src/AWS.Deloy.CLI` is set as the start up project.
+* Go to `AWS.Deploy.CLI debug properties` by clicking the following drop down
+
+![](debug-properties.png)
```

Comment:
Since this is public, I think it's worth the extra ~60 seconds to draw a better looking arrow in Paint or PowerPoint.

Reply:
Done